src/Supportive/DependencyInjection/ServiceContainerBuilder.php

```diff
@@ -129,10 +129,17 @@ private static function loadServices(ContainerBuilder $container, ?SplFileInfo $
             return;
         }
 
-        if (!file_exists($cacheFile->getPathname())
+        $dirname = $cacheFile->getPath() ?: '.';
+
+        if ((
+            !is_dir($dirname)
+            && mkdir($dirname.'/', 0777, true)
+            && !is_dir($dirname)
```

Comment:
This logic is similar to: https://github.com/qossmic/deptrac/blob/20c23c1299db2f6efd7d35c0a28b376b55bf56a6/src/Supportive/OutputFormatter/BaselineOutputFormatter.php#L45